docs/community/maintainer/release-flow.md

```diff
@@ -74,10 +74,16 @@ Replace URLs with appropriate ones.
 
 Example: https://github.com/aquasecurity/trivy/releases/tag/v0.52.0
 
+### Merging the auto-generated Helm chart update PR
+Once the release PR is merged, there will be an auto-generated PR that bumps the Trivy version for the Trivy Helm Chart. An example can be seen [here](https://github.com/aquasecurity/trivy/pull/8638).
```

Comment:
I thought:
let's add a `note` that tests can be broken.
after `release` workflow is complete - you need to re-run the failing tests.

Reply:
Ah yes you're right - added https://github.com/aquasecurity/trivy/pull/8640/files#diff-e647f776c72c91393790e0855ba2c391d33d0c917903e7a67d8740e6eeecd5a4R80-R82